src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -1067,14 +1067,14 @@ internal bool MethodGroupIsCompatibleWithDelegate(BoundExpression receiverOpt, b
             TypeSymbol destination,
             DiagnosticBag diagnostics)
         {
-            Debug.Assert(sourceValue != null);
+            RoslynDebug.Assert(sourceValue != null);
             Debug.Assert(!sourceValue.IsBad);
 
             SpecialType destinationType;
             if ((object)destination != null && destination.IsEnumType())
```

Comment:
>destination [](start = 24, length = 11)

Should `destination` be declared as nullable?

Reply:
The only callers pass non-null.

---
In reply to: [367184280](https://github.com/dotnet/roslyn/pull/40946#discussion_r367184280) [](ancestors = 367184280)